app/persistence/message/leader-election.go

```diff
@@ -0,0 +1,13 @@
+package message
```

Comment:
I think that this code is not related to `persistence` in any way. It is the business logic of the consensus-message handler, thus it should be in that package imo